python/ray/autoscaler/command_runner.py

```diff
@@ -535,3 +539,37 @@ def _docker_expand_user(self, string, any_char=False):
                 return string.replace("~", self.home_dir, 1)
 
         return string
+
+    def run_init(self, as_head):
+        image = self.docker_config.get("image")
+        if image is None:
+            image = self.docker_config.get(
+                "{}_image".format("head" if as_head else "worker"))
+
+        self._check_docker_installed()
+        if self.docker_config.get("pull_before_run", True):
+            assert image, "Image must be included"
```

Comment:
can you provide a more elaborate error message?

Reply:
Yep!